Overlays für die Karten 1908 - 1999 hinzugefügt

HanoldM opened the pull request:
Overlays hinzugefügt,
!!!!ABER Koordinatensystem für
1936, 1963, 1977,1987, 1999 ist unterschiedlich zu UFK und 1908

Versuch die Koordinaten auf die gleiche Form über QGIS Reproduktion hat nicht funktioniert.

HanoldM committed:
Overlays für die Karten 1908 - 1999 hinzugefügt

Overlays hinzugefügt,
!!!!ABER Koordinatensystem für
1936, 1963, 1977,1987, 1999 ist unterschiedlich zu UFK und 1908

HanoldM commented:
Ok, das hab ich jetzt auch so gemacht. 
Problem ist jetzt, dass zwar das Koordinatensystem richtig angezeigt wird, aber die Koordinaten aber immer noch falsch sind. 
Wenn ich dann trotzdem versuche die Datei im Script einzubinden geht wieder garnichts beim testen. 
Hab hier mal die beiden .js Dateien angehängt. Einmal die Transformierte Version von 1936 und einmal die "normale" richtige Version von 1908.
![Bildschirmfoto 2020-01-25 um 09 36 33](https://user-images.githubusercontent.com/57889928/73118827-d962ee00-3f59-11ea-9ee5-7220e5c6e464.png)

AndyBraun commented:
Also bevor man ein Shapefile (mit der von Christian gezeigten Methode) in einem neuen Koordinatenbezugssystem exportieren kann, muss man sicher gehen, dass das **aktuelle** KBS in den Layer Properties richtig zugeordnet ist. 
**Vor** dem Exportieren muss also in den Layer Properties UTM ausgewählt werden. Erst dann kann man exportieren (dort WGS84) angeben, sodass die Meter-Koordinaten (515605,5366912) in geographische Koordinaten (9.1806,48.3204) transformiert werden.

HanoldM committed:
Karten von 1936 - 1999 korrigiert

- Karten mit vormals falscher Projektion korrigiert und eingefügt
- Hinzufügen der korrigierten geoJSON und js Datein
- Legende Aktualisiert
-(Falsche geoJSON und js datein entfernen) eventuell

sommergeo committed:
Merge branch 'master' into Markus
